index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adopt a Dog</title>
+    <script src="js/dogs.js" charset="utf-8"></script>
+    <link href="styles/style.css" type="text/css" rel="stylesheet">
+</head>
+
+<body>
+
+    <header>
+        <div class="header-left">
+            <img class="logo" src="images/adoptadog-logo.svg" alt="adoptadog logo">
+            <h1>Seattle’s premiere dog adoption agency</h1>
+        </div>
+        <div class="header-right">
+            <div class="pay">
+                <img class="cart" src="images/cart-icon.svg" alt="shopping cart">
+                <p>$1234.56</p>
+            </div>
+            <button>Checkout</button>
+        </div>
+    </header>
+
+    <nav>
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li>/</li>
+            <li><a href="dogs.html">Dogs</a></li>
+            <li>/</li>
+            <li><a href="blog.html">Blog</a></li>
+
+        </ul>
+    </nav>
+
+    <section class="adopt">
+
+        <img class="col-1 row-1" src="images/home-intro.jpg" alt="Man hugging a dog">
+
+        <div class="col-2 row-1">
+            <h2>Adopt A Dog</h2>
+
+            <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem
+                quidustrum fuga. Faceaquae
+                estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum
+                haritatur
+                arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus
+                am aut amet ant fugiatum, utem non reptat ad magnienecto volorent fugia nonsedi gendand itatquos as
+                magnimpor aut molor autatia ad est officimus accae et omnia cus, od qui ate</p>
+        </div>
+
+    </section>
+
+    <section class="dogs">
+
+        <article class="row-1 col-1">
```

Comment:
When shrinking down screen size, your dogs become too small to read. I would suggest utilizing minmax() and autofit to create flexible column sizes. This allows you to create a grid that responds well to any screen size! Check out this article on it here: https://mastery.games/post/tile-layouts/#minmax-Has-your-Back

You will notice this method will also eliminate the need for your row/col classes (row-1, col-1, row-2, col-2...) classes.